src/Neo/Network/P2P/Payloads/TransactionAttributes/Conflicts.cs

```diff
@@ -13,10 +13,12 @@
 using Neo.Json;
 using Neo.Persistence;
 using Neo.SmartContract.Native;
+using System;
 using System.IO;
 
 namespace Neo.Network.P2P.Payloads
 {
+    [Obsolete("Use ConflictsAttribute instead")]
     public class Conflicts : TransactionAttribute
```

Comment:
Before remove maybe is easier to mantain if we share the code, 

```suggestion
    public class Conflicts : ConflictsAttribute 
```

Reply:
@shargon thats what i said here https://github.com/neo-project/neo/pull/3488#issuecomment-2350933572

Reply:
> Before remove maybe is easier to mantain if we share the code

But code is already there, keep this pr  this way as much easier isn't it~~~~